gapic-generator-java/pom.xml

```diff
@@ -14,7 +14,7 @@
     <clirr.skip>true</clirr.skip>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <!-- Should be in sync with repositories.bzl -->
-    <googleapis.commit>44d6bef0ca6db8bba3fb324c8186e694bcc4829c</googleapis.commit>
+    <googleapis.commit>9fcfbea0aa5b50fa22e190faceb073d74504172b</googleapis.commit>
```

Comment:
This is probably a tech debt on our end. `gapic_metadata.proto` should be updated automatically along side other "config protos" like [service.proto](https://github.com/googleapis/sdk-platform-java/blob/main/java-common-protos/proto-google-common-protos/src/main/proto/google/api/service.proto).

Reply:
Yep, agreed. I will leave this here for now, but LMK if you want me to file a tracking issue with some details.